docs/api-markdown-documenter/index.js

```diff
@@ -4,16 +4,30 @@
  */
 
 const chalk = require("chalk");
-
+const versions = require("../data/versions.json");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-renderApiDocumentation().then(
-	() => {
-		console.log(chalk.green("API docs written!"));
-		process.exit(0);
-	},
-	(error) => {
-		console.error("API docs could not be written due to an error:", error);
-		process.exit(1);
-	},
-);
+const renderMultiVersion = process.argv[2];
```

Comment:
Let's add some top-level documentation here explaining what this script does and, in particular, what arguments are expected and what they mean.

Reply:
Added